server/http.go

```diff
@@ -127,6 +131,20 @@ func handleHTTPRequest(p *Plugin, w http.ResponseWriter, r *http.Request) (int,
 	case "/debug/vars":
 		goexpvar.Handler().ServeHTTP(w, r)
 		return 0, nil
+
+	// Workflow
+	case routeWorkflowRegister:
+		{
+			if c.SourcePluginId != "" {
```

Comment:
Why `!= ""`?

Reply:
Means "Any plugin".

Reply:
Understood, but (a) why any plugin, as opposed to strictly "workflow", and if not strictly "workflow" - why not let other properly authorized sysadmin user?) requests through? (b) maybe warrants a more intention-revealing packaging? an `isAuthorized` function or something of the sort? (no comments! :) )